newdle/client/src/Calendar.js

```diff
@@ -0,0 +1,46 @@
+import 'react-dates/initialize';
+import React, {useCallback, useEffect} from 'react';
+import {HTML5_FMT} from 'moment';
+import {useDispatch, useSelector} from 'react-redux';
+import {DayPickerSingleDateController as DayPicker} from 'react-dates';
+import {getCalendarDates, getCalendarActiveDate} from './selectors';
+import {setActiveDate, clearCalendarActiveDate} from './actions';
+import {serializeDate, toMoment} from './util/date';
+
+import 'react-dates/lib/css/_datepicker.css';
+import styles from './Calendar.module.scss';
+
+export default function Calendar() {
+  const calendarDates = useSelector(getCalendarDates);
+  const activeDate = useSelector(getCalendarActiveDate);
+  const dispatch = useDispatch();
+
+  const onDateChange = useCallback(date => dispatch(setActiveDate(serializeDate(date))), [
```

Comment:
i think the standard naming style for event handlers in react is `handle...`, i.e. `handleDateChange`